<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobUi</class>
 <widget class="QWidget" name="JobUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(190, 242, 255);
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>546</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_Name">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Field">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>filed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_closed">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#ff0000;&quot;&gt;The comany is not any more employeeing for this job&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_prifile">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Company prifile</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Company">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Company</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Skils :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_skils">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_WorkplaceType">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>workplacve type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Location">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>locatio</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Time">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_salary">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>salaru</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_TimeCreated">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>time created</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_apply">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(20,176,191);
</string>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
